invenio_rdm_records/contrib/imprint/custom_fields.py

```diff
@@ -53,13 +55,14 @@ def mapping(self):
                 "isbn": {"type": "keyword"},
                 "pages": {"type": "keyword"},
                 "place": {"type": "keyword"},
+                "edition": {"type": "keyword"},
             },
         }
 
 
 IMPRINT_NAMESPACE = {
     # Imprint
-    "imprint": None,
+    "imprint": "",
```

Comment:
was the change to mutable value done on purpose?

Reply:
Good catch. Nope, just my copy of the code was older. I've fixed it.